config/initializers/config.rb

```diff
@@ -1,31 +1,15 @@
 Config.setup do |config|
   # Name of the constant exposing loaded settings
   config.const_name = 'Settings'
+  config.use_env = true
+  config.env_prefix = 'SETTINGS'
+  config.env_separator = '__'
```

Comment:
is the double underscore deliberate?

Reply:
Yes, it's to allow users to use underscores in env vars (if they need to?!) while allowing config to split a double underscore into dotted strings e.g. test__value becomes `Settings.test.value`, while test_value would remain `Settings.test_value`